app/src/androidTest/java/com/pajato/android/gamechat/FABTest.java

```diff
@@ -0,0 +1,131 @@
+package com.pajato.android.gamechat;
+
+import android.app.Activity;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.pajato.android.gamechat.main.MainActivity;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.swipeLeft;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withTagValue;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * All tests are based on the the following documentation:
+ * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class FABTest {
+
+    @Rule public ActivityTestRule<MainActivity> mRule = new ActivityTestRule<>(MainActivity.class);
+    private String spaceValue;
+
+    /**
```

Comment:
I've taken to using /** Some short description. */ and @Before public void ... to achieve more conciseness.  These are both acceptable practices under the Android and Google coding conventions documents.